packages/palette/src/elements/Modal/ModalBase.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useRef } from "react"
+import { createPortal } from "react-dom"
+import { FocusOn } from "react-focus-on"
+import styled, { css } from "styled-components"
+import { Flex, FlexProps } from "../Flex"
+
+const ModalWrapper = styled(Flex)<{ overlay?: boolean }>`
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  /* TODO: Use a token? Or some other alternative to wildstyle z-index? */
+  z-index: 9999;
+
+  ${({ overlay }) =>
+    overlay &&
+    css`
+      /* TODO: Use a token? */
+      background-color: rgba(229, 229, 229, 0.5);
```

Comment:
I know a lot of the stuff in the modal looks pre-tokens? So maybe there's a helper for this sort of thing?

Reply:
Well, some of the things (drop shadows, etc) may not be specified in tokens. Not that they shouldn't be, just something we haven't standardized yet.